.travis.yml

```
@@ -20,7 +20,15 @@ matrix:
           env: TEST_CONFIG="phpunit-integration-legacy-solr.xml" SOLR_VERSION="7.7.2"  CORES_SETUP="single" SOLR_CORES="collection1"
         - php: 7.1
           env: TEST_CONFIG="phpunit-integration-legacy-solr.xml" SOLR_VERSION="7.7.2"  CORES_SETUP="cloud" SOLR_CLOUD="yes"
-
+        # Solr 8.5.2
+        - php: 7.1
+          env: TEST_CONFIG="phpunit-integration-legacy-solr.xml" SOLR_VERSION="8.5.2"  CORES_SETUP="dedicated"
+        - php: 7.1
+          env: TEST_CONFIG="phpunit-integration-legacy-solr.xml" SOLR_VERSION="8.5.2"  CORES_SETUP="shared"
+        - php: 7.1
+          env: TEST_CONFIG="phpunit-integration-legacy-solr.xml" SOLR_VERSION="8.5.2"  CORES_SETUP="single" SOLR_CORES="collection1"
+        - php: 7.1
+          env: TEST_CONFIG="phpunit-integration-legacy-solr.xml" SOLR_VERSION="8.5.2"  CORES_SETUP="cloud" SOLR_CLOUD="yes"
```

Suggestion to make sure we test versions provided by P.sh to make sure we are doing cloud first approach:

```suggestion
        # Solr 8
        - php: 7.1
          env: TEST_CONFIG="phpunit-integration-legacy-solr.xml" SOLR_VERSION="8.5.2"  CORES_SETUP="dedicated"
        - php: 7.1
          env: TEST_CONFIG="phpunit-integration-legacy-solr.xml" SOLR_VERSION="8.4.1"  CORES_SETUP="shared"
        - php: 7.1
          env: TEST_CONFIG="phpunit-integration-legacy-solr.xml" SOLR_VERSION="8.4.1"  CORES_SETUP="single" SOLR_CORES="collection1"
        - php: 7.1
          env: TEST_CONFIG="phpunit-integration-legacy-solr.xml" SOLR_VERSION="8.5.2"  CORES_SETUP="cloud" SOLR_CLOUD="yes"
```

Ref: https://docs.platform.sh/configuration/services/solr.html